master/buildbot/newsfragments/_getSshDataPath.bugfix

```diff
@@ -0,0 +1 @@
+:py:class:`~buildbot.util.GitStepMixin`: Fix GitStepMixin._getSshDataPath() to safeTranslate the builder's name (:issue:`5949`)
```

Comment:
The readers of the release notes will not know what GitStepMixin or safeTranslate are. I think better idea would be to say something along the following "git steps will now ensure that invalid characters are not used in temporary file names"

Reply:
Ok, sure

Reply:
Should I keep the class reference?

Reply:
I think it's not necessary. Unless it's a specific step most users reading the release notes won't know exactly what it is.

Reply:
Okie